go/kbfs/libkbfs/keybase_service_util.go

```diff
@@ -59,6 +60,13 @@ func setupDiskBlockCache(ctx context.Context, config Config, username string) {
 				"for disk block cache: %+v", err)
 		}
 	}
+
+	// Send to Favorites cache
+	info := homeTLFInfo{
+		PublicTeamID:  *publicHandle.toFavToAdd(false).Data.TeamID,
+		PrivateTeamID: *publicHandle.toFavToAdd(false).Data.TeamID,
```

Comment:
Also we probably shouldn't blindly make calls on `publicHandle` or `privateHandle` here, since they could be `nil` if there was an error above.  Also also, I'm not sure about blindly dereferencing the TeamIDs either, can they ever legitimately be nil?